.ci/install_crio.sh

```diff
@@ -55,23 +55,20 @@ then
 		else
 			crio_version=$(get_version "externals.crio.meta.openshift")
 		fi
+		crictl_version=$(get_test_version "externals.critools.openShift3Version")
+		crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/v$crictl_version/crictl-$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
 	else
 		crio_version=$(get_version "externals.crio.version")
+		crictl_version=$(get_test_version "externals.critools.version")
+		crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/$crictl_version/crictl-$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
```

Comment:
These two `crictl_url` variables *look* identical but on closer inspection one contains a `v` as a version prefix and the other doesn't. Is that really correct?

If it is, please could you make this clearer by reworking the code to something like:

```sh
version_prefix=""

if [ ... ]; then
    version_prefix="v"
    # ...
else
   # ...
fi

cri_tools_site=https://github.com/kubernetes-sigs/cri-tools
cri_tools_file="crictl-${crictl_version}-linux-$(${cidir}/kata-arch.sh -g).tar.gz"

crictl_url="${cri_tools_site}/releases/download/${version_prefix}${crictl_version}/${cri_tools_file}
```
```

Reply:
Yes, I adapt it as suggested.